src/storybook/__snapshots__/Stories.test.ts.snap

```diff
@@ -111442,7 +111442,7 @@ exports[`Storyshots Views / Orders / Order details cancelled 1`] = `
                     <div
                       class="MuiTypography-root-id OrderCardTitle-cardHeader-id MuiTypography-body1-id"
                     >
-                      Unfulfilled (1)
+                      Canceled (1)
```

Comment:
Please check if it's expected change

Reply:
Yes. There was a bug and canceled fulfillments were shown as unfulfilled.

Reply:
Okay 👍